src/modules/services_manager_service.rb

```diff
@@ -393,7 +385,7 @@ def exportable_enabled_services
     #
     # @return [Hash{String => SystemService}]
     def exportable_disabled_services
-      services.select { |service_name, service| service.changed? && !enabled(service_name) }
+      services.select { |name, service| service.changed? && !enabled(name) }
```

Comment:
maybe we should also here filter out static services as we cannot manipulate with it.

Reply:
I am just retaining the original behaviour here. And, for SLE 15 SP1, we will be able to manipulate them.